RadixWallet/Features/SettingsFeature/Preferences/Preferences+View.swift

```diff
@@ -68,7 +68,7 @@ extension Preferences.View {
 					#endif
 				}
 			}
-			.background(Color.app.gray4)
+			.background(Color.app.gray5)
```

Comment:
Background colors in Settings views were incorrect, so updated it in every screen.